modules/network-observability-filtering-ebpf-rule.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-filtering-ebpf-rule_{context}"]
+= Filtering eBPF flow data with rules
+You can configure the `FlowCollector` to filter eBPF flows using a global rule as a method to control the flow of packets cached in eBPF's flows table based on certain configuration.
+
+.Procedure
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. Under the *Provided APIs* heading for *Network Observability*, select *Flow Collector*.
+. Select *cluster* then select the *YAML* tab.
+. Configure the `FlowCollector` custom resource. A few sample configurations are as follows:
++
+
+[%collapsible]
+.Filter Kubernetes service traffic to specific POD IP endpoint
+====
+[source, yaml]
+----
+apiVersion: flows.netobserv.io/v1beta2
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  namespace: netobserv
+  deploymentModel: Direct
+  agent:
+    type: eBPF
+    ebpf:
+      metrics:
+        enable: true    <1>
+      flowFilter:
+        action: Accept  <2>
+        cidr: 172.210.150.1/24 <3>
+        protocol: SCTP
+        direction: Ingress
+        destPortRange: 80-100
+        peerIP: 10.10.10.10
```

Comment:
u add `enable: true` here to enable flowfilter

Reply:
OK, I also added enable to the Required configuration table: https://github.com/openshift/openshift-docs/pull/74622/files#diff-c1f389cfa44881b2885e5709e478c820d4f98f140dbb208b231639b47ec4d159R15

Reply:
yeah better to have enable: true even in the YAML